src/OrchardCore.Modules/OrchardCore.Facebook/Manifest.cs

```diff
@@ -39,3 +40,10 @@
         "OrchardCore.Recipes.Core",
     }
 )]
+
+[assembly: Feature(
+    Id = FacebookConstants.Features.Pixel,
+    Name = "Facebook Pixel",
+    Category = "Facebook",
+    Description = "Provides a way to enable Facebook Pixel tracking for your site."
```

Comment:
It's called Meta Pixel now. I wouldn't rename the whole feature or anything but at least:

```suggestion
    Description = "Provides a way to enable Meta (Facebook) Pixel tracking for your site."
```

Reply:
I renamed all the Facebook features to Meta. The module and classes are still called Facebook to prevent any breaking change.